src/screens/Game/SectionSummary/SectionSummary.tsx

```diff
@@ -1,21 +1,222 @@
+        paddingBottom: "3%",
+        paddingLeft: "4%",
+        paddingRight: "4%",
+      }}
+    >
+      <View
+        style={{ display: "flex", flexDirection: "row", alignItems: "center" }}
+      >
+        <Text
+          style={{
+            color: "#2B3674",
+            fontSize: 16,
+            fontStyle: "normal",
+            fontWeight: "700",
+            paddingBottom: "2%",
+          }}
+        >
+          {title}
+        </Text>
+      </View>
+      <Text
+        style={{
+          color: `${statColor}`,
+          fontSize: 36,
+          fontStyle: "normal",
+          fontWeight: "700",
         }}
+      >
+        {stat}
+      </Text>
+    </View>
+  );
+}
+
+interface TitleComponentProps {
+  iconName: string;
+  iconColor: string;
+  title: string;
+}
+
+function TitleComponent({ iconName, iconColor, title }: TitleComponentProps) {
+  return (
+    <View
+      style={{
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "space-between",
+        backgroundColor: "#FFF",
+        paddingTop: "2%",
+        paddingBottom: "2%",
+        paddingLeft: "4%",
+        paddingRight: "4%",
+      }}
+    >
+      <View
+        style={{ display: "flex", flexDirection: "row", alignItems: "center" }}
+      >
+        <View
+          style={{
+            width: 46,
+            height: 46,
+            borderRadius: 46,
+            backgroundColor: "#F4F7FE",
+            justifyContent: "center",
+            alignItems: "center",
+          }}
+        >
+          {<FontAwesome5 name={iconName} size={24} color={iconColor} />}
+        </View>
+        <Text
+          style={{
+            color: "#2B3674",
+            fontSize: 16,
+            fontStyle: "normal",
+            fontWeight: "700",
+            paddingLeft: "3%",
+          }}
+        >
+          {title}
+        </Text>
+      </View>
+    </View>
+  );
+}
+
+export default function SectionSummary({ route }: Props) {
+  const subject = "subject" in route.params ? route.params.subject : null;
+
+  const gameDetails = useSelector<
+    RootState,
+    GameDetails["lastSessionsMetrics"][0] | null
+  >((state) => {
+    const { lastSessionsMetrics } = state.game;
+    if (lastSessionsMetrics && lastSessionsMetrics.length > 0) {
+      return lastSessionsMetrics[0];
+    }
+    return null;
+  });
+
+  const subjectDetails =
+    subject === "math" ||
+    subject === "trivia" ||
+    subject === "writing" ||
+    subject === "reading"
+      ? gameDetails[subject]
+      : null;
+
+  const questions = 10;
+  const timePer = 70;
+
+  let icon = "square-root-alt";
+  let color = "#EA4335";
+
+  if (subject === "reading") {
+    icon = "book-open";
+    color = "#FE7D35";
+  } else if (subject === "writing") {
+    icon = "file-alt";
+    color = "#A066FF";
+  } else if (subject === "trivia") {
+    icon = "question-circle";
+    color = "#34BC99";
+  }
+
+  return subjectDetails.attempted ? (
```

Comment:
Could you remove the header, and add the continue button at then end to return back to the main page? We want to be as accurate as possible to the figma. Use src/components/ContinueButton